regexes/device/mobiles.yml

```diff
@@ -4204,6 +4247,14 @@ Infinix:
     - regex: 'Infinix[ _-]([a-z0-9_\-]+)'
       model: '$1'
 
+# InnJoo
+InnJoo:
+  regex: 'InnJoo'
+  device: 'smartphone'
+  models:
+    - regex: 'X'
+      model: 'X'
```

Comment:
This detection seems to be a bit to unrestrictive. It matches any `X` in the user agent as long as it also contains `InnJoo`. Would be better to check for `InnJoo[ _]X`. That way it would also match this ua: `UCWEB/2.0 (MIDP-2.0;U;Adr 5.1;en-US;InnJoo_X) U2/1.0.0 UCBrowser/10.9.5.983 U2/1.0.0 Mobile`